.github/ISSUE_TEMPLATE/module-rewrite.md

```diff
@@ -10,7 +10,7 @@ assignees: ""
 Module(s):
 Week(s):
 
-Rewriting a week in the curriculum will often involve updating different sections of the curriculum, including the backlog and workshops. You can use this issue to organise the different changes in the curriculum content for a week at CYF.
+Rewriting a week in the curriculum will often involve updating different sections of the curriculum, including the backlog and workshops. You can use this issue to organise the different changes in the curriculum content for a week at <!-- CYF-ONLY -->CYF<!-- END-CYF-ONLY -->.
```

Comment:
All these comments should be removed right? As it makes sense to keep CYF when it comes to contributions

Reply:
I don't think there are any remaining in the common content, only in our own org content.